h2/src/main/org/h2/mvstore/db/ValueDataType.java

```diff
@@ -22,6 +22,7 @@
 import org.h2.mvstore.WriteBuffer;
 import org.h2.mvstore.rtree.SpatialDataType;
 import org.h2.mvstore.rtree.SpatialKey;
+import org.h2.mvstore.type.BasicDataType;
```

Comment:
`DataType` import in the line below is not used and there is a couple of unneeded casts in lines 685 and 699.